packages/react/src/components/Sidebar/SidebarItem/SidebarItem.tsx

```diff
@@ -467,10 +468,22 @@ export const SidebarItem: OverridableComponent<SidebarItemTypeMap> = (inProps: S
           {!!children &&
             React.Children.map(children, (child: any, idx: number) => {
               // eslint-disable-next-line @typescript-eslint/no-unused-vars
-              const { text, inset, ...rest } = child.props;
+              const { text, inset, onClick, ...rest } = child.props;
 
               return (
-                <SidebarItemTooltipItem key={idx} className={clsx(classes.tooltipItem)} tabIndex={idx} {...rest}>
+                <SidebarItemTooltipItem
+                  key={idx}
+                  className={clsx(classes.tooltipItem)}
+                  tabIndex={idx}
+                  {...rest}
+                  onClick={() => {
+                    onClick();
```

Comment:
```suggestion
                    onClick?.(event);
```
Может отсутствовать и тогда крашнется